app/views/orgs/lti_configurations/show.html.erb

```diff
@@ -3,77 +3,72 @@
 
   <% lms_name = current_lti_configuration.lms_name(default_name: "your Learning Management System") %>
 
-  <div class="columns">
-    <div class="one-fourth column">
-      <%= render partial: 'organizations/side_menu' %>
+  <div class="site-content">
+    <div class="site-content-cap">
+      <h2 class="site-content-heading">Connect to <%= lms_name %></h2>
     </div>
-    <div class="three-fourths column">
-      <div class="site-content">
-        <div class="site-content-cap">
-          <h2 class="site-content-heading">Connect to <%= lms_name %></h2>
-        </div>
 
-        <div class="site-content-body markdown-body">
-          <h3>Connect GitHub Classroom to <%= lms_name %></h3>
-          <p>You or your system administrator will need to add a new application to
-          <%= lms_name %> with the following credentials generated below
-          by GitHub Classroom.</p>
+    <div class="site-content-body markdown-body">
+      <h3>Connect GitHub Classroom to <%= lms_name %></h3>
+      <p>You or your system administrator will need to add a new application to
+      <%= lms_name %> with the following credentials generated below
+      by GitHub Classroom. Check out our
+      <%= link_to "step-by-step guide for configuing GitHub Classroom with #{lms_name}.", "https://www.youtube.com/watch?v=dQw4w9WgXcQ" %>
```

Comment:
Very soon this document will redirect to the relevant support docs for configuring GitHub Classroom within an LMS, but in the meantime it redirects [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ) ;)